### PR TITLE
(#2786, #2769, #2783) Infographic tweaks and analytics

### LINKED ISSUE #2769: Add custom analytics tracking
# As an analytics analyst, I want to be able embed specific tracking codes on the view and enlarge functionality, so I can track their use 

> **ESTIMATE** NEEDS ESTIMATE

### Acceptance Criteria

  ```gherkin
  Scenario: Click events for “View and Print Infographic” from the page
    Given user is viewing page infographic
    When user clicks on "View and Print Infographic" button
    And page click request is sent
    Then the following parameters should be captured
      | parameter | value |
      | event71   |       |
      | prop66   |     viewprint|inline|imageTitle  |
      | prop67   |       |

  ```
### Resources

* Feature: https://tracker.nci.nih.gov/browse/CGOV-11460
* Mockups: https://publicis.invisionapp.com/share/CJW8VVAV87U
* Web Page: https://www.cancer.gov/research/areas/biology

### Notes

* See Enabler: Update Analytics Handler to push to NCIDataLayer #124
* Will apply to infographics in-line at medium and large formats, as well as infographic page
* "imageTitle" in the analytics should reference the content in the "Name" field of the infographic media type in the CMS
* Original analytics
  * 
```
  On click of “View and Print Infographic” from the page we would like to pass:
	  	1.Event71 (infographic click)
	  	2.Prop66 to collect viewprint|inline|image title
	  	3.Prop67 to collect the pagename
```
	
	

---

### LINKED ISSUE #2783: View & Print Link on 768px and less should not show on all Infographic templates
### Issue description
"View & Print Infographic" link is showing up on 768px and less on all templates (inline, 40% and infographic page)

> **ESTIMATE** 

### Steps to reproduce the issue

1.  Go to https://ncigovcdode340.prod.acquia-sites.com/test/immune-system
2. Shrink browser to 768px to 320px

### What's the expected result?
"View & Print Infographic" link appears

### What's the actual result?
"View & Print Infographic" link should not appear

```gherkin
Scenario: Web site visitor views an Infographic on a page on Tablet or Mobile
  Given a user is viewing a page with an infographic inline at 100% or 40%
    And the browser width is 768px or less
  Then the "View and Print Infographic" link should not display
```

```gherkin
Scenario: Web site visitor views an Infographic Page on Tablet or Mobile
  Given a user is viewing an infographic page
    And the browser width is 768px or less
  Then the "View and Print Infographic" link should not display
```

### Additional details / screenshot
Screen Shot 2020-05-08 at 12 32 00 PM

### LINKED ISSUE #2786: Update Spanish text link for Infographics
### Issue description
Spanish translation for "View and Print Infographic" is incorrect.

> **ESTIMATE** 

### What's the expected change?
Link should display "VER E IMPRIMIR INFOGRAFÍA"

### What's the current functionality?
Link displays as "VER Y IMPRIMIR INFOGRAFÍA"

### What's the updated acceptance criteria?
```gherkin
Scenario: Infographic Spanish Link on all templates
Given that language is set to "ES"
Then the user will see "VER E IMPRIMIR INFOGRAFÍA"
```

### Additional details / screenshot
Go to: http://ncigovcdode345.prod.acquia-sites.com/espanol/test/infographic-sprint2-demo-page-es

### Related Tickets
* Original Story Ticket #9999
* Original Story Ticket #9999

### PR DESCRIPTION
**Pull request by adriancofie**
Closes #2786 
Closes #2769 
Closes #2783

ODE: 

http://ncigovcdode346.prod.acquia-sites.com/test/embed-info-med
http://ncigovcdode346.prod.acquia-sites.com/test/embed-info-lrg

### CODE REVIEW

**Comment by claireblaustein** [comment]
@adriancofie View/Print Infographic link still showing at mobile breaks on infographic page: 

http://ncigovcdode346.prod.acquia-sites.com/about-nci/organization/screen-to-save-infographic

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/analytics/analytics.js [hunk 1]
```diff
@@ -725,4 +725,13 @@ $(window).on('load',function(){
 
         NCIAnalytics.ProfilePanelLinkClick($this, linkText, name);
     });
+
+    // Tracks clicks of "View and Print" link on infographic items.
+    $('.infographic-link a').on('click', function() {
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/analytics/analytics.js:730]: I think you want your selector to be `a.infographic-link`...

**Comment by kate-mashkina** [comment]
Analytics:
1. Full page infographics are missing the title of the infographic in prop66:
http://ncigovcdode346.prod.acquia-sites.com/about-nci/organization/nci-at-a-glance


Screen Shot 2020-05-20 at 10 41 46 AM

**Comment by claireblaustein** [comment]
@adriancofie 
One back end weirdness - when putting the infographic inline, the preview shows some code and the infographic title instead of view/print


image

No other additional issues.

**Comment by adriancofie** [comment]
- [ ] View Infographic link -- add to full page view

- [ ] Prop 66 on Infographic  full page

- [ ]  View Infographic link --  in cotent editor

**Comment by claireblaustein** [comment]
@adriancofie All tests passed. Looking good!

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/media/infographic/infographic_macros.twig [hunk 2]
```diff
@@ -44,16 +44,14 @@
...
         <figcaption>
            {% if content.field_caption|render|striptags|trim %}
             <div class="caption-container no-resize">
                 {{ content.field_caption }}
             </div>
           {% endif %}
-          <div class="caption-container">
-            <a href="{{enlarge_uri}}" class="infographic-link" target="_blank">{{ link_text|trans }}</a>
+          <div class="caption-container infographic-link-container">
+            <a href="{{enlarge_uri}}" class="infographic-link" data-link_title= "{{ content.name|field_value }}" target="_blank">{{ link_text|trans }}</a>
```
[the remark this answers is not in the document]
Comment (bryanpizzillo) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/media/infographic/infographic_macros.twig:54]: The quotes get encoded, so not an issue.

**Pull request merged**